pkg/controller/pagerdutyintegration/pagerdutyintegration_controller.go

```diff
@@ -32,16 +33,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
 const (
 	controllerName = "pagerdutyintegration"
 )
 
+var fedramp = os.Getenv("FEDRAMP") == "true"
```

Comment:
Why does this default to "true"? It is inconsistent with the template, which I agree with as we should default to current production behavior.

```suggestion
var fedramp = os.Getenv("FEDRAMP") == "false"
```

Reply:
i would argue that because unless the environment variable `FEDRAMP` equals `true` exactly, then the variable `fedramp` actually defaults to false. we can reverse the logic though, if you'd like

Reply:
Depends on how you want to parse the env var, it doesn't have to be the literal string `"true"`, you could support `"True"` and `"TRUE"` but that's an implementation detail.  Default behavior needs to be that it's treated as not(true) when whatever true string is not set.

Reply:
if the value of the env variable `FEDRAMP` is `true`, the comparison will evaluate to true, otherwise it will evaluate to false. either way, it will set the value of the go variable `fedramp` to the result of that comparison

i think it's already doing what you want (case-insensitivity aside)... what am i missing?

Reply:
:facepalm: yup I was reading it wrong.. thanks!